src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -33,6 +32,13 @@ const DraggableComponent: FC<draggableProps> = ({
   title = '',
 }) => {
   const draggableIds = useContext(disableDrag);
+  const [draggableImage, setDraggableImage] = useState<string>('');
+
+  useEffect(() => {
+      getAssigneeProfilePic(draggableId).then((res) => {
+        setDraggableImage(res);
```

Comment:
Here please check for error and response, also is response a url?

Reply:
yes response is URL